projects/angular/src/utils/_mixins.scss

```
@@ -36,9 +36,9 @@ $clr-outline-spread: variables.$clr_baselineRem_2px;
   outline-offset: variables.$clr_baselineRem_1px;
 }
 
-@mixin include-slim-outline-style-form-fields() {
-  outline: Highlight solid variables.$clr_baselineRem_2px;
-  outline: -webkit-focus-ring-color solid variables.$clr_baselineRem_2px;
+@mixin include-slim-outline-style-form-fields($width: variables.$clr_baselineRem_2px) {
+  outline: Highlight solid $width;
+  outline-color: -webkit-focus-ring-color;
 }
```

We should use cds token and separate the properties the way we wanted due to the issue found with custom properties.
```suggestion
@mixin include-slim-outline-style-form-fields($width: tokens.$cds-global-space-2) {
  outline-color: Highlight;
  outline-color: -webkit-focus-ring-color;
  outline-style: solid;
  outline-width: $width;
}
```

---

Corrected in commit 59b8c82